### PR TITLE
Trees update (+ processings)

### PR DESCRIPTION
**Pull request by Nunien**
linked to https://github.com/fli-iam/shanoir-ng/issues/525

### CODE REVIEW

**Comment by jcomedouteau** [comment]
Just correct the processing display name

File: shanoir-ng-front/src/app/subjects/tree/subject-tree.component.html [hunk 1]
```diff
@@ -19,9 +19,14 @@
             <node [label]="examination | subjectExaminationLabel" awesome="fas fa-diagnoses" (labelClick)="showExaminationDetails(examination.id)" class="examination-node">
                 <node *ngFor="let acquisition of examination.datasetAcquisitions" [label]="acquisition.name" awesome="fas fa-laptop-medical" class="acquisition-node">
                     <node *ngFor="let dataset of acquisition.datasets" [label]="dataset.name" awesome="fas fa-brain" (labelClick)="showDatasetDetails(dataset.id)" class="dataset-node">
+                        <node *ngFor="let processing of dataset.processings" [label]="processing.type" awesome="fas fa-cog">
+                            <node *ngFor="let dataset of processing.outputDatasets" [label]="dataset.name" awesome="fas fa-brain" (labelClick)="showDatasetDetails(dataset.id)" class="dataset-node">
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/subjects/tree/subject-tree.component.html:23]: Don't re-use dataset variable name ?

**Review by michaelkain — CHANGES_REQUESTED** [review]
Hi Julien, we agreed on not again having big pull requests. Could you please split them?

**Comment by michaelkain** [comment]
Hi Julien, could you please take care of the conflicts, that we can merge it for Thursday? Thank you, Michael

**Comment by Nunien** [comment]
@michaelkain  Done !

**Pull request merged**